### PR TITLE
Include ConfigMaps from all controller namespaces in bugreport

### PR DESCRIPTION
**Pull request by tiffanny29631**
b/362134244

### CODE REVIEW

File: pkg/bugreport/bugreport.go [hunk 1]
```diff
@@ -314,25 +314,31 @@ func (b *BugReporter) fetchResources(ctx context.Context, gv schema.GroupVersion
...
+	var namespaces = []string{
+		configmanagement.ControllerNamespace,
+		configmanagement.RGControllerNamespace,
+		configmanagement.MonitoringNamespace,
+	}
+	for _, ns := range namespaces {
+		configMapList, err := b.clientSet.CoreV1().ConfigMaps(ns).List(ctx, metav1.ListOptions{})
+		if err != nil {
+			b.ErrorList = append(b.ErrorList, fmt.Errorf("failed to list %s configmaps: %v", ns, err))
+		} else {
```
Comment (karlkfi) [inline pkg/bugreport/bugreport.go:329]: you can use continue instead of else here to reduce cyclomatic complexity

Comment (tiffanny29631) [reply pkg/bugreport/bugreport.go]: Done

**Review by karlkfi — APPROVED** [review]
/lgtm

**Pull request merged**